### PR TITLE
Schedule icon and Refresh Social

### PR DESCRIPTION
**Pull request by javipacheco**
This ticket covers:

- Add pull to refresh to Social: you can get new tweets from social screen
- Schedule icon in the navigation bar: new icon in Schedule for go to event quickly
- SBT Idea plugin added

@fedefernandez can you please review? Thanks!

### CODE REVIEW

File: src/main/scala/com/fortysevendeg/android/scaladays/ui/qrcode/Layout.scala [hunk 1]
```diff
@@ -16,20 +16,43 @@
...
-import macroid.ActivityContextWrapper
+import macroid.{Ui, ActivityContextWrapper}
 
 trait Layout
   extends Styles {
 
-  var scanButton = slot[Button]
+  self : Fragment with AnalyticsServicesComponent =>
+
+  val defaultDisplayMs: Long = 0L
```
Comment (fedefernandez) [inline src/main/scala/com/fortysevendeg/android/scaladays/ui/qrcode/Layout.scala]: Constants would be upper CamelCase, isn't it?

**Comment by fedefernandez** [comment]
My only concern is that most of `analyticsServices.sendEvent` calls are inside `Ui` blocks. Is that necessary @javipacheco?

**Comment by fedefernandez** [comment]
On the other hand, the code os much more functional, like too much the refactor over `Ui` blocks

File: src/main/scala/com/fortysevendeg/android/scaladays/ui/qrcode/Layout.scala (hunk 1 above)
Comment (javipacheco) [inline src/main/scala/com/fortysevendeg/android/scaladays/ui/qrcode/Layout.scala]: it's right

**Comment by javipacheco** [comment]
In your first question, `On.click` need a `Ui` block. I'll try to use `uiStartActivity`in macroid-extras. I don't know if I'll have some problem with that

**Comment by fedefernandez** [comment]
I see, LGTM!

**Pull request merged**